docs/how-tos/set-configuration.md

```diff
@@ -58,5 +58,4 @@ config = Config.from_file(config_path)
 ```
 
 !!! note
-
-    In the Python API, the `Rag.chat()` function also allows you to set certain RAG-specific configurations like `document` and `assistants`.
+    In the Python API, the [`Rag.chat()` function](../references/python-api.md#ragna.core.Chat) also allows you to set certain RAG-specific configurations like `document` and `assistants`.
```

Comment:
```suggestion

    In the Python API, the [`Rag.chat()` function](../references/python-api.md#ragna.core.Chat) also allows you to set certain RAG-specific configurations like `document` and `assistants`.
```

Reply:
dang ok did not expect `prettier` to do that. I thought that might be because [`proseWrap: always`](https://github.com/Quansight/ragna/blob/cfd3a85121ce27209a20f522963379d678db4199/.prettierrc.yaml#L1) is set, but changing that did not change the `prettier` error/formatting for this